thorlcr/thorutil/thbuf.cpp

```diff
@@ -1536,7 +1533,8 @@ class CSharedWriteAheadDisk : public CSharedWriteAheadBase
         freeChunks.kill();
         freeChunksSized.kill();
         highOffset = 0;
-        spillFileIO->setSize(0);
+        tempFileIO->setSize(0);
+        tempFileOwner->noteSize(0);
```

Comment:
I don't know why this (the setSize(0)) was implemented this way.
I think it should just delete the file, and let it be recreated if it was needed.

(not a change for now in this PR)

Reply:
Jira to look into this later: https://hpccsystems.atlassian.net/browse/HPCC-31985